docs/guides/misc/client_errors.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Known Client Errors
+---
+
+Documentation for various known errors in the client.
+
+## store_list
+
+### STORE ERROR: account is too new
+
+New accounts may only make purchases via ((%Cstore_list%)) after 3 days.
+
+### STORE ERROR: unknown
+
+This error is typically indicative of an untrusted Steam account. Unfortunately, the reason for this error is often difficult to dianogse.
+
+- The Steam account usually must exist for at least 30 days
+- The Steam account needs to have purchased at least $5 of things on Steam, from a valid payment method
+- Accounts occasionally seem to be 'blacklisted' by Steam. If this happens, the account cannot ever be trusted
+  - It is not clear if VAC/game bans count for this
+  - Steam support has historically not assisted in this matter. Creation of a new account is the only known solution thus far
```

Comment:
Once a new account is created, please contact hackmud support by joining the discord and creating a *support* ticket. We will assist you in re-associating your hackmud account with the new steam account.

(or something)

We should document this on the wiki too-- porting the 'this is how you report issues' content.

Reply:
> We should document this on the wiki too-- porting the 'this is how you report issues' content.

but this is a different work item to tackle